test/e2e/apimachinery/get_server_version.go

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Under `test/e2e/` e2e test filenames don't contain verb.
It is nice to rename this to just `server_version.go` without `get_`.

Reply:
Thanks for the catch. Fixed now